raiden/network/transport/udp/udp_transport.py

```diff
@@ -436,6 +436,14 @@ def send_async(
                 message=message,
             )
 
+    def send_global(
+            self,
+            room: str,
+            message: Message,
+    ) -> None:
+        """ This method exists only for interface compatibility with MatrixTransport """
+        self.log.warning('UDP has no global room mechanism. Ignoring.')
```

Comment:
This message seems more of a comparison against the matrix. I would suggest to have this message be something like:
```
UDP is unable to send global messages. Ignoring
```

Reply:
Done.